charts/testkube-api/templates/role.yaml

```diff
@@ -676,6 +681,15 @@ rules:
       - create
       - delete
       - deletecollection
+  - apiGroups:
+      - ""
+    resources:
+      - events
```

Comment:
events can be created

Reply:
Oh, I wasn't aware that we are creating events, thanks, will add 👍

Reply:
Actually, should it be there? Will the jobs create events? Apparently there is `events-role-{namespace}` assigned to `testkube-api-server` itself

Reply:
Ok, so basically as in this PR 👍